packages/react-app/src/App.jsx

```diff
@@ -33,6 +34,7 @@ import { BuilderListView, ChallengeDetailView, ProfileView, SignInView } from ".
 */
 
 // const serverUrl = "https://backend.ether.delivery:49832/"
+// TODO PR COMMENT: we should probably remove the trailing / here. So we build urls doing ${serverUrl}/foo/bar instead of ${serverUrl}foo/bar
 const serverUrl = "http://localhost:49832/";
```

Comment:
I think we should remove the trailing `/` here. That way we can build urls in other places doing `${serverUrl}/foo/bar` instead of `${serverUrl}foo/bar`, which feels more natural.

Not having trailing `/` is the default in other projects that I've worked on, as a trailing `/` will usually cause some urls like `foo.com/bar//baz` and things break.

If you agree I can create a new issue to tackle this. It should be really simple, but we need to track down any place where `serverUrl` is used.

Reply:
I agree :D

Reply:
This will be tracked at #36. Resolving here.